collections/guides/own_and_use_us_bank_accounts/index.md

```diff
@@ -0,0 +1,135 @@
+    width: 1200
+    height: 630
+---
+# Use U.S. Bank Accounts without touching them
+{: .no_toc }
+
+<span class="base-alert success">
+  <span>
+    This guide will quickly enable you to be compliant with new Nacha Encryption rules. 
+  </span>
+</span>
+
+Safely storing bank account data in a way that enables your entire organization to safely use, is no small feat. It takes a strong encryption pattern, including a KMS, key rotation, multiple encryption keys, access controls, and so much more. 
+
+In this guide, we will show you how to quickly collect and secure your customers' U.S. Bank Account information with Basis Theory. You'll learn how to collect the data using our Elements, toknize the information, and use our Serverless Reactor platform to communicate with Spreedly to obtain a Payment Method. 
+
+If you'd like to follow along with this guide jump right into our sample app - <a href="https://codesandbox.io/s/github/Basis-Theory/basis-theory-js-examples/tree/master/?from-embed=&file=/bank/public/index.html">Find it here!</a>
+
+### Table of contents
+{: .no_toc .text-delta }
+1. 
+{:toc}
+
+## Add Elements to your website
+
+<span class="base-alert warning">
+  <span>
+    To start, you'll need a new Elements Application with the <code>token:bank:create</code> permission
+  </span>
+</span>
+
+First, add two new `div` tag elements to your HTML. These will serve as a place to attach two separate Element tags for both Routing Number and Bank Account.
+
+```html
+    <div class="row row-input" id="routing_number"></div>
+    <div class="row row-input" id="account_number"></div>
+```
+
+Second, you'll need to add the following javascript code to your application to mount each element:
+
+```js 
+  await BasisTheory.init(ELEMENTS_KEY, {
+    elements: true,
+  });
+
+  let routingNumber = BasisTheory.elements.create('text', {
+    ...options,
+    targetId: 'routingNumber',
+    placeholder: 'Routing Number',
+    'aria-label': 'Routing Number',
+  });
+  let accountNumber = BasisTheory.elements.create('text', {
+    ...options,
+    targetId: 'accountNumber',
+    placeholder: 'Account Number',
+    'aria-label': 'Account Number',
+  });
+
+  await routingNumber.mount('#routing_number');
+  await accountNumber.mount('#account_number');
+```
+
+Finally, you'll see two new Elements appear on your page. Next, we will tokenize the data entered into them!
+
+## Create an Atomic Bank from your new Elements
+This is where our Elements shine, you'll interact with our SDK as if you're passing raw text. 
+
+To create an [Atomic Bank](https://docs.basistheory.com/api-reference/#atomic-banks), you'll just need to pass the Element references from Step 1 into the `atomicBank.create` method.
+
+```js
+const token = await BasisTheory.elements.atomicBank.create({
+      bank: {
+        routingNumber,
+        accountNumber,
+      },
+    });
+```
+
+Once completed, you'll have a new Atomic Bank assigned to `token`.

```

Comment:
Maybe mention that this token/tokenId is then typically stored by your application to later use this bank data, eg. through our serverless reactor platform.